### DISCUSSION TITLE
01/20 오프라인 미팅

### DISCUSSION THREAD

**Discussion by fkdl0048** [BRIDGE-DEV/Unity_Custom_Editor_Study#2] [General]
매주 금요일에 진행하는 온라인 회의를 01/20 오프라인으로 대체  

오후 7시 광화문역

현재 진행예정인 프로젝트의 기능들을 다양한 레퍼런스등을 참고해서 이야기해볼 것

**Comment by fkdl0048** [comment]
금요일 전까지 보내주신 링크나 다른 예제들 좀 더 알아보고 정리해서 가겠읍니다..!

**Comment by kugorang** [comment]
홍콩익스프레스 본점에서 저녁 먹기로 했습니다

https://naver.me/5s9zOQPL
